x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/node_available_warning/node_available_warning.tsx

```diff
@@ -9,58 +9,58 @@ import React, { Fragment, FC } from 'react';
 import { EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { mlNodesAvailable, permissionToViewMlNodeCount } from '../../../../ml_nodes_check';
-import { cloudDeploymentId, isCloud } from '../../../../jobs/new_job_new/utils/new_job_defaults';
+import { cloudDeploymentId, isCloud } from '../../../../services/ml_server_info';
 
 export const NodeAvailableWarning: FC = () => {
   if (mlNodesAvailable() === true || permissionToViewMlNodeCount() === false) {
     return null;
-  } else {
-    const id = cloudDeploymentId();
-    return (
-      <Fragment>
-        <EuiCallOut
-          title={
-            <FormattedMessage
-              id="xpack.ml.jobsList.nodeAvailableWarning.noMLNodesAvailableTitle"
-              defaultMessage="No ML nodes available"
-            />
-          }
-          color="warning"
-          iconType="alert"
-        >
-          <p>
-            <FormattedMessage
-              id="xpack.ml.jobsList.nodeAvailableWarning.noMLNodesAvailableDescription"
-              defaultMessage="There are no ML nodes available."
-            />
-            <br />
-            <FormattedMessage
-              id="xpack.ml.jobsList.nodeAvailableWarning.unavailableCreateOrRunJobsDescription"
-              defaultMessage="You will not be able to create or run jobs."
-            />
-            {isCloud && id !== null && (
-              <Fragment>
-                <br />
-                <FormattedMessage
-                  id="xpack.ml.jobsList.nodeAvailableWarning.linkToCloudDescription"
-                  defaultMessage="Please edit your {link}. You may enable a free 1GB machine learning node or expand your existing ML configuration."
-                  values={{
-                    link: (
-                      <EuiLink href={`https://cloud.elastic.co/deployments?q=${id}`}>
-                        <FormattedMessage
-                          id="xpack.ml.jobsList.nodeAvailableWarning.linkToCloud.hereLinkText"
-                          defaultMessage="Elastic Cloud deployment"
-                        />
-                      </EuiLink>
-                    ),
-                  }}
-                />
-              </Fragment>
-            )}
-          </p>
-        </EuiCallOut>
-        <EuiSpacer size="m" />
-      </Fragment>
-    );
   }
+
+  const id = cloudDeploymentId();
```

Comment:
`getCloudDeploymentId`? 👀

Reply:
i purposely left this change as every function in this service except `loadMlServerInfo` is a getter and changing one would mean i'd need to change all and that would be a lot of churn.
Does anyone else have an opinion on this? I don't mind changing all, it'll just touch a lot of files.

Reply:
added getter renaming changes to this PR as all files which use the functions have already been touched for the include path change